### ISSUE TITLE
Dist file contains 'const'

### ISSUE THREAD

**Issue by maplemike** [bebraw/react-pagify#16]
The built dist file (for version 0.11) contains const declarations. I don't know where all browsers are in terms of compatibility but in Safari it produces:

```
SyntaxError: Unexpected keyword 'const'. Const declarations are not supported in strict mode.
```

https://github.com/bebraw/react-pagify/blob/263bff6703eca5794190bc96b2b68681a04b552a/dist/react-pagify.js#L225

**Comment by bebraw** [comment]
Thanks for the heads up. I'll get that fixed. There's likely some little Babel tweak for that.

**Comment by bebraw** [comment]
It's a dependency issue. I think I'll fix this with context rework (#12). While investigating the issue, I modernized the library build so it wasn't in vain.

**Comment by bebraw** [comment]
Fixed now given I pretty much rewrote the implementation. There's no direct dependency on *segmentize* anymore and the issue has been fixed there anyhow.
